C7FIT/User.swift

```diff
@@ -13,24 +13,51 @@ import Firebase
     A model representation of a User
  */
 struct User {
-//    let uid: String
     let email: String
-//    let name: String?
-//    let weight: Int?
-//    let height: Int?
-//    let bmi: Double?
-//    let mileTime: String?
-
-    init(authData: FIRUser) {
-//        uid = authData.uid
-        email = authData.email!
-    }
+    let photoURL: String?
+    let name: String?
+    let bio: String?
+    let weight: String?
```

Comment:
Ohh man I wish I knew about this earlier! I think I may stick to keeping the values as Strings for now though and focus on refactoring/refining the other components of this app. Will look into this in the future.